pipeline.py

```diff
@@ -197,5 +200,14 @@ def main(result_dir: str, data_atlas_dir: str, data_train_dir: str, data_test_di
         help='Directory with testing data.'
     )
 
+    feature_extraction_params = {
```

Comment:
Extracted outside the `main` function to be able to run the `run_pipeline.py` with configurable settings.